scifi.d/misc/wwz/index.html

```diff
@@ -1,202 +1,212 @@
 <!doctype html>
 <html class="no-js" lang="">
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="x-ua-compatible" content="ie=edge">
-        <title>World War 102: Worlds at war</title>
-        <meta name="description" content="">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link rel="stylesheet" href="/style.css">
-        <link rel="apple-touch-icon" href="/apple-touch-icon.png">
-        <!-- Place favicon.ico in the root directory -->
-        <style>
-         table, th, td {
-           border: 1px solid white;
-           padding: 10px;
-         }
-         td { text-align: center; }
-         td:first-of-type { text-align: left; }
-        </style>
-    </head>
-    <body>
-        <!--[if lt IE 8]>
-            <p class="browserupgrade">
-            You are using an <strong>outdated</strong> browser. Please
-            <a href="http://browsehappy.com/">upgrade your browser</a> to improve
-            your experience.
-            </p>
-        <![endif]-->
-<h1> World war 102: Entire Worlds at War </h1>
-<i>the war has been going for almost a century! untold billions died! Help! I cannot wake up from this nightmare! I cannot wake up!</i>
-<br>
-<br>
-<i>Do you think Satan or God is afraid of the Old Ones?</i> <br>
--a quote from a random soldier in a trench, before getting shot in the head.
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>World War 102: Worlds at war</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="apple-touch-icon" href="/apple-touch-icon.png">
+    <!-- Place favicon.ico in the root directory -->
+    <style>
+    table, th, td {
+      border: 1px solid white;
+      padding: 10px;
+    }
+    td { text-align: center; }
+    td:first-of-type { text-align: left; }
+    </style>
+  </head>
+  <body>
+    <!--[if lt IE 8]>
+<p class="browserupgrade">
+You are using an <strong>outdated</strong> browser. Please
+<a href="http://browsehappy.com/">upgrade your browser</a> to improve
+your experience.
+</p>
+<![endif]-->
+    <header>
+      <h1> World war 102: Entire Worlds at War </h1>
+      <i>the war has been going for almost a century! untold billions died! Help! I cannot wake up from this nightmare! I cannot wake up!</i>
+      <i>Do you think Satan or God is afraid of the Old Ones?</i> <br>
+        -a quote from a random soldier in a trench, before getting shot in the head.
+        G
```

Comment:
why is there a stray "G"

Reply:
https://www.youtube.com/clip/Ugkxuz1emzc6sTCMifULO8v9mBx_D-Xc9wHc

Reply:
(i have no idea it wasn't intentional)

Reply:
YOU DARE USING MY OWN ~~MEMES~~ AGAINST ME POTTER?

Reply:
nom nom nom